helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -193,10 +193,13 @@ private fun Collection<TextLocation>.groupByText(baseDir: File): List<TextLocati
 }
 
 private fun TextLocation.resolve(baseDir: File): String? {
-    val lines = baseDir.resolve(path).readText().lines()
-    return if (lines.size <= endLine) {
-        null
-    } else {
-        lines.subList(startLine - 1, endLine).joinToString(separator = "\n")
-    }
+    if (!baseDir.isDirectory) return null
```

Comment:
You could drop this separate check, because if `baseDir` is not a directory then also `file` does not resolve to a file.

Reply:
Good catch! Done.